layouts/index.html

```diff
@@ -9,7 +9,7 @@
         {{ end }}
 
         <div class="posts-list">
-          {{ $pag := .Paginate (where .Data.Pages "Type" "post") }}
+          {{ $pag := .Paginate (where .Site.RegularPages "Type" "post") }}
```

Comment:
It's better to use [`mainSections`](https://gohugo.io/functions/where/#mainsections) to avoid hard-coding things.

Reply:
Excellent suggestion @VincentTam . Can you squash commits when merging or do you want me create a new branch for you.

Reply:
Oops.  I should have committed my changes on top of 79abb2e.  Thanks for changing your code.